src/app/core/entity/schema-datatypes/datatype-month.ts

```diff
@@ -28,6 +28,9 @@ import { EntitySchemaDatatype } from "../schema/entity-schema-datatype";
  */
 export const monthEntitySchemaDatatype: EntitySchemaDatatype = {
   name: "month",
+  // Property schemas should have `additional: "YYYY-MM"` to properly display the month
+  viewComponent: "DisplayDate",
```

Comment:
DisplayDate should ensure we display it only year + month (or otherwise have a separate display component for this?)